Controller/SitemapController.php

```diff
@@ -78,8 +86,8 @@ public function indexAction($_format)
         }
 
         $urls = $this->urlProvider->getUrlInformation();
```

Comment:
i would put the urlProvider into the configuration as well. then i can have one provider for each sitemap.